src/supertux/menu/editor_level_menu.cpp

```diff
@@ -0,0 +1,50 @@
+//  SuperTux
+//  Copyright (C) 2016 Hume2 <teratux.mail@gmail.com>
+//
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#include "supertux/menu/editor_level_menu.hpp"
+
+#include "gui/menu.hpp"
+#include "gui/menu_item.hpp"
+#include "gui/menu_manager.hpp"
+#include "editor/editor.hpp"
+#include "supertux/menu/menu_storage.hpp"
+#include "supertux/menu/options_menu.hpp"
+#include "supertux/level.hpp"
+#include "supertux/screen_manager.hpp"
+#include "util/gettext.hpp"
+#include "video/color.hpp"
+
+EditorLevelMenu::EditorLevelMenu()
+{
+  add_label(_("Level properties"));
+  add_hl();
+  add_textfield(_("Name"), &(Editor::current()->get_level()->name));
```

Comment:
auto level = Editor::current()->get_level();

and then use that in the add_* calls.